### PR TITLE
Bumped thecodingmachine/safe version from 1.3 to 2.4

### PR DESCRIPTION
**Pull request by leondirectvps**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /.idea/
 clover.xml
 .phpunit.result.cache
+infection.log
```
Comment (hyperized) [inline .gitignore:5]: I don't think this belongs in the PR 👍🏻

Comment (leondirectvps) [reply .gitignore]: You're right. Updated.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 /vendor/
 /.idea/
 clover.xml
-.phpunit.result.cache
+.phpunit.result.cache
```
Comment (hyperized) [inline .gitignore:4]: 👋🏻 This still appears to be a change that should not be in the PR. Could you make sure to not commit any changes to this file?

**Comment by leondirectvps** [comment]
I only just realized this change would require to drop PHP 7.4 support. It's now EOL anyways, but it's out of this PR's scope...

**Comment by hyperized** [comment]
@leondirectvps Taking PHP 7.4 out of this project requires a few more changes. I also ran into some issues with 8.0 support already. I'm proposing to close this PR and replace it with: https://github.com/hyperized/hostfact/pull/49. Would that be acceptable?

**Comment by leondirectvps** [comment]
@hyperized absolutely! Thanks for you efforts

**Pull request closed without merging**